app/src/main/java/com/libs/facerecognition/FacePreprocessor.java

```diff
@@ -26,7 +26,7 @@
 public class FacePreprocessor {
     public final String Tag = "ImagePreProcessor";
     private final FaceDetector faceDetector;
-
+    private final int FaceMargin = 10;
```

Comment:
I tested this locally and I fell like this margin is way to high. Couldn't it be 0?

Reply:
+1